src/main/webapp/app/tasks/task-search/task-search.component.html

```diff
@@ -0,0 +1,6 @@
+<form [formGroup]="taskSearch">
+  <mat-form-field class="search-bar">
+    <input (keyup)="searchQuery($event)" matInput placeholder="Search tasks">
```

Comment:
need formControlName="queryInput" here when trying to add validation - it is missing and therefore broken now